tests/continuous/test_flip_flop.py

```diff
@@ -286,3 +287,82 @@ def test_encompassing_sector_size_raises(data, dims, skipna):
     """
     with pytest.raises(DimensionError):
         encompassing_sector_size(data, dims, skipna=skipna)
+
```

Comment:
Need to add test to check that it works with dask. In fact we need to update the tests for the `flip_flop_index` to check that it works with dask. See https://github.com/nci/scores/blob/develop/tests/continuous/test_standard.py#L468 for an example for testing with dask.